info-ui/src/main/java/com/example/info_ui/AdditionalInfoScreenFragment.kt

```diff
@@ -1,70 +1,275 @@
-            LinearLayoutManager(context, LinearLayoutManager.VERTICAL, false)
-        binding.recyclerForecastDetailed.adapter =
-            ForecastDetailsAdapter(forecast, requireContext())
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center
+    ) {
+        Text(text = day, fontSize = 24.sp)
+        ForecastDetailsList(forecastList)
     }
-    private fun handleEvent(event: AdditionalInfoScreenViewModel.Event) {
-        when (event) {
-            is AdditionalInfoScreenViewModel.Event.OnBack -> {
-                findNavController().popBackStack()
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SingleForecastItem(
+    day: ForecastItem
+) {
+    val temperature =
+        "${day.main.temp.toBigDecimal().setScale(0, RoundingMode.HALF_UP).toInt()}\u00B0"
+    val icon = "https://openweathermap.org/img/wn/${day.weather[0].icon}@2x.png"
+    val description = day.weather[0].description
+    val wind = day.wind.speed
+    val humidity = day.main.humidity
+    val pressure = day.main.pressure
+    Card(
+        modifier = Modifier
+            .fillMaxWidth()
+            .wrapContentHeight()
+            .padding(8.dp),
+        shape = RoundedCornerShape(8.dp),
+        elevation = CardDefaults.cardElevation(defaultElevation = 8.dp)
+    ) {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .wrapContentHeight(),
+            horizontalArrangement = Arrangement.SpaceBetween
+        ) {
+            var hour = day.date.removeRange(0, 11)
+            hour = hour.removeRange(5, 8)
+            Text(
+                text = hour,
+                fontSize = 24.sp,
+                modifier = Modifier.padding(top = 16.dp, start = 16.dp)
+            )
+            Row(
+                modifier = Modifier
+                    .wrapContentSize()
+            ) {
+                Text(text = temperature, fontSize = 24.sp, modifier = Modifier.padding(top = 16.dp))
+                Image(
+                    modifier = Modifier
+                        .width(64.dp)
+                        .height(64.dp),
+                    painter = rememberAsyncImagePainter(model = icon),
+                    contentDescription = null,
+                )
+            }
+        }
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .wrapContentHeight(),
+            horizontalArrangement = Arrangement.SpaceBetween
+        ) {
+            Text(text = description, fontSize = 16.sp, modifier = Modifier.padding(start = 16.dp))
+            Row(
+                modifier = Modifier
+                    .wrapContentSize()
+                    .padding(end = 16.dp)
+            ) {
+                Icon(
+                    painter = painterResource(id = R.drawable.ic_wind),
+                    contentDescription = null,
+                )
```

Comment:
add space between wind icon and wind speed value

Reply:
also use alignments when you have icon and text nearby. they should have common centre line

Reply:
extract composable function for icon+text for wind, humidity and pressure